app/src/main/java/com/example/myapplication/fragment/InputDataFragment.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.myapplication.fragment
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.lifecycle.lifecycleScope
+import androidx.room.Room
+import com.example.myapplication.Database.Book
+import com.example.myapplication.Database.BookDatabase
+import com.example.myapplication.MainActivity
+import com.example.myapplication.R
+import com.example.myapplication.databinding.FragmentInputDataBinding
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+
+class InputDataFragment : Fragment() {
+
+    private var _binding: FragmentInputDataBinding? = null
+    private val binding get() = requireNotNull(_binding)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
скорее всего здесь будет warning, за таким нужно следить
```suggestion
    ): View {
```